readthedocsext/theme/templates/projects/import_config.html

```diff
@@ -5,12 +5,16 @@
   {% trans "Add a configuration file to your project" %}
 {% endblock project_add_subheader %}
 
-{% block project_add_content_classes %}ui fourteen wide tablet twelve wide computer column{% endblock %}
+{% block project_add_content_classes %}
+  ui fourteen wide tablet twelve wide computer column
+{% endblock project_add_content_classes %}
 
 {% block project_add_content_form %}
   <div class="ui small info message">
     {% blocktrans trimmed %}
-      Save and commit the following example configuration to your repository root as <code>.readthedocs.yaml</code>.
+      Read the Docs <strong>requires a <code>.readthedocs.yaml</code> file at the root of the repository</strong> to build your project's documentation.
+      Open a pull request in your repository with this file using the following example for your documentation tool, commit, push and merge the changes.
+      Then, come back to this page and finish the process.
```

Comment:
Summarizing everything above, I feel it works best clear and short. Mentioning the file is required is a great upgrade, as is setting the expectation we're waiting on the user for something.

```suggestion
      A <strong><code>.readthedocs.yaml</code> configuration file is required at the root of your repository</strong>.
      Save and commit this file before continuing.
```